models/cow_protocol/ethereum/cow_protocol_ethereum_trades.sql

```diff
@@ -28,7 +30,7 @@ trades_with_prices AS (
            feeAmount                 as fee_amount,
            ps.price                  as sell_price,
            pb.price                  as buy_price
-    FROM {{ source('gnosis_protocol_v2_ethereum', 'GPv2Settlement_evt_Trade') }}
+    FROM {{ source('gnosis_protocol_v2_ethereum', 'GPv2Settlement_evt_Trade') }} settlement
```

Comment:
I could do that here. Not sure I understand the decoded sources comment, could you elaborate?

Reply:
Yeah sorry, so basically we have to set up incremental filters using this syntaxt
`{% if is_incremental() %}  x.blocktime >= date_trunc("day", now() - interval '1 week') {% endif %}`

It allows to speed up incremental runs a lot because it's only looking to append/merge data for the last week instead of scanning the entire table.

My comment was about you including more filters like these, on sources (e.g., `{{ source('gnosis_protocol_v2_ethereum', 'GPv2Settlement_evt_Trade') }}`), prices, etc...

Reply:
If you look at this opensea spell, we apply incremental filters a lot on different downstream dependencies https://github.com/duneanalytics/spellbook/blob/main/models/opensea/ethereum/opensea_v1_ethereum_events.sql

Reply:
Ok I have added several incremental filters here (found lots of places). I hope it still compiles.